src/pages/settings/Wallet/WalletPage/BaseWalletPage.js

```diff
@@ -102,19 +101,15 @@ function BaseWalletPage(props) {
     }, [paymentMethod.selectedPaymentMethod.bankAccountID, paymentMethod.selectedPaymentMethod.fundID, paymentMethod.selectedPaymentMethodType]);
 
     const resetSelectedPaymentMethodData = useCallback(() => {
-        // The below state values are used by payment method modals and we reset them while closing the modals.
-        // We should only reset the values when the modal animation is completed and so using InteractionManager.runAfterInteractions which fires after all animaitons are complete
```

Comment:
Perhaps some of this comment is still valid and could be moved to the place that we call InteractionManager?

Reply:
Ah, I scanned the file but didn't get that contect. Thanks